### ISSUE TITLE
.net core and ServerManager.OpenRemote() doesn't work

### ISSUE THREAD

**Issue by TNMephaGroup** [dotnet/core#4845]
referring to an issue: https://github.com/dotnet/core/issues/2691 as the original issue got closed without any resolution or milestones

I can confirm that Microsoft.Web.Administration ver 11.0.0 still has the same issue with nulled values. We have thousands of VMs to manage and I can not use the proposed solution by deploying the API on all VMs

**Comment by padmasrid** [comment]
@bariscaglar, any idea?

**Comment by Sam--Shan** [comment]
guys,maybe you should try to refer the C:\Windows\System32\inetsrv\Microsoft.Web.Administration.dll ,It's a .net framework version and it works.
check this
https://forums.iis.net/t/1236991.aspx?Micorsoft+Web+Administration+Remote+administration+failure

**Comment by mitchello** [comment]
Any news on this issue ?

**Comment by VirtualTim** [comment]
I don't think this is the right repository for this issue. I'm pretty sure this is a bug in the closed-source Microsoft.Web.Administration.
